tests/Frontend.test.cpp

```diff
@@ -1146,4 +1146,39 @@ TEST_CASE_FIXTURE(BuiltinsFixture, "module_scope_check")
     CHECK_EQ(toString(ty), "number");
 }
 
+TEST_CASE_FIXTURE(BuiltinsFixture, "use_correct_global_scope")
```

Comment:
Let me know if this isn't the right place for the test case.

Had to reimplement a few functions specifically for autocomplete: not sure if they should be implemented on Fixture instead